spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -165,20 +168,29 @@ public void destroy() {
         ShutdownHook.getInstance().destroyAll();
     }
 
-    private void initClient() {
+    public void setInitTmClient(boolean initTmClient) {
+        this.initTmClient = initTmClient;
+    }
+
+    public void initClient() {
         if (LOGGER.isInfoEnabled()) {
             LOGGER.info("Initializing Global Transaction Clients ... ");
         }
         if (StringUtils.isNullOrEmpty(applicationId) || StringUtils.isNullOrEmpty(txServiceGroup)) {
             throw new IllegalArgumentException(
                 "applicationId: " + applicationId + ", txServiceGroup: " + txServiceGroup);
         }
-        //init TM
-        TMClient.init(applicationId, txServiceGroup);
-        if (LOGGER.isInfoEnabled()) {
-            LOGGER.info(
-                "Transaction Manager Client is initialized. applicationId[" + applicationId + "] txServiceGroup["
-                    + txServiceGroup + "]");
+
+        // Not every service needs to initialize tm client.
+        // Need to initialize tm client when use @GlobalTransactional.
```

Comment:
and @GlobalLock.

Reply:
ok,thanks for your reviewed.

Reply:
> and @GlobalLock.
@xingfudeshi  Why?

Reply:
Sorry.My mistake.Only @GlobalTransactional need to initialize tm client.

Reply:
Then you can't just use existsAnnotation Method,you have to create new one check for @GlobalTransactional only.

Reply:
> ok,thanks for your reviewed.

please review again.

Reply:
> Sorry.My mistake.Only @GlobalTransactional need to initialize tm client.

thank you, please review again.